test/functional/admin/document_collection_group_memberships_controller_test.rb

```diff
@@ -56,14 +57,15 @@ def move_params
   end
 
   test 'DELETE #destroy moves documents and redirects when Move clicked' do
```

Comment:
Should we change references to 'documents' to 'memberships'?

Reply:
Good point 👍